README.md

```diff
@@ -54,13 +61,28 @@ The general layout of this repo looks like this:
 Each test subdirectory will automatically have its own local Atlas deployment. As a result, database and collection names will not conflict between different AI/ML integrations. To connect to your local Atlas using a connection string, `utils.sh` has a `fetch_local_atlas_uri` that you can call from the `run.sh` script within your subdirectory. For example:
 
 ```bash
-. $workdir/src/.evergreen/utils.sh
+. .evergreen/utils.sh
 
 CONN_STRING=$(fetch_local_atlas_uri)
 ```
 
 Stores the local Atlas URI within the `CONN_STRING` var. The script can then pass `CONN_STRING` as an environment variable to the test suite.
 
+#### Running tests locally.
+
+We can run the tests with a local checkout of the repo.
+
+For example, to run the `docarray` tests using local atlas:
+
+```bash
+export DIR=docarray
+bash .evergreen/fetch-repo.sh
+bash .evergreen/provision-atlas.sh
+bash .evergreen/execute-tests.sh
+```
+
+Use `.evergreen/setup-remote.sh` instead of `.evergreen/provision-atlas.sh` to test against the remote cluster.
```

Comment:
nit: what if there was _yet another_ script just to execute these steps and it took the DIR as a command line argument?

Reply:
These steps will often need to be done independently, so I don't like the idea of a single wrapper script.